.python-version

```diff
@@ -0,0 +1 @@
+2.7.14
```

Comment:
[node-gyp](https://github.com/nodejs/node-gyp) doesn't support Python 3.x.x.

Reply:
[Travis CI](https://travis-ci.org/) supports [pyenv](https://github.com/pyenv/pyenv). I like it.